crypto/x509v3/v3_cpols.c

```diff
@@ -413,7 +413,8 @@ static void print_notice(BIO *out, USERNOTICE *notice, int indent)
             num = sk_ASN1_INTEGER_value(ref->noticenos, i);
             if (i)
                 BIO_puts(out, ", ");
-            tmp = i2s_ASN1_INTEGER(NULL, num);
+            if ((tmp = i2s_ASN1_INTEGER(NULL, num)) == NULL)
+                return;
```

Comment:
Just break out?  That will just result in some *very* weird printout, quite incomprehensible and hard to understand.  I would *strongly* prefer something like this:

```
            if ((tmp = i2s_ASN1_INTEGER(NULL, num)) == NULL)
                BIO_puts(out, "InvalINT");
            else
                BIO_puts(out, tmp);
```